src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -112,7 +112,7 @@ def create_keyvault(client, resource_group_name, vault_name, location, #pylint:d
                                              object_id=object_id,
                                              permissions=permissions)]
     properties = VaultProperties(tenant_id=tenant_id,
-                                 sku=Sku(name=sku, family='A'),
```

Comment:
It's been removed and can't be set anywhere else.
There was a 'bug' in the SDK that exposed family as a parameter when the only valid value for it was 'A'.